_maps/configs/sv_bluntsmoken.json

```diff
@@ -0,0 +1,13 @@
+{
+	"map_name": "SV Bluntsmoken",
+	"map_short_name": "Weed ship",
```

Comment:
follow `[X]-class [Purpose] Vessel` name format for `map_name` and `[X]-class` name format for `map_short_name`